consensus/src/runway/mod.rs

```diff
@@ -289,7 +289,7 @@ where
                 }
                 Request::NewestUnit(salt) => {
                     trace!(target: "AlephBFT-runway", "{:?} Newest unit request received {:?}.", self.index(), salt);
-                    self.on_request_newest(node_id, salt).await
+                    self.on_request_newest(node_id, salt)
```

Comment:
on_unit_message can be make non-async check also if this can propagate more

Reply:
done